Apps/AdminWebClient/src/ClientApp/.eslintrc.js

```diff
@@ -10,6 +10,7 @@ module.exports = {
         "@vue/typescript/recommended",
         "@vue/prettier",
         "@vue/prettier/@typescript-eslint",
+        "plugin:sonarjs/recommended",
```

Comment:
not sure this is needed.

Reply:
it implements the sonarjs rules as the "sonar way" quality profile used by sonarcloud so it shows some code smells as warnings in our code.

Reply:
yeah, but from the looks of it is just adding an additional linter that is competing with the other ones. The rules look similar enough

Reply:
I guess my concern is that it might override other recommendations coming from vue/typescript settings, and I am not sure they would be better